orchestration/ptycho_jobscript.py

```diff
@@ -0,0 +1,169 @@
+import os
+from datetime import datetime
+from collections import OrderedDict
+
+default_path_client_id = "/global/software/ptycholive-dev/nersc_api_keys/clientid.txt"
+default_path_private_key = "/global/software/ptycholive-dev/nersc_api_keys/priv_key.pem"
+default_path_job_script = "/global/software/ptycholive-dev/ACME_Data_Cleaning_And_Assembly/src/acme_data_cleaning/nersc_job_script"
+default_path_ptychocam_nersc = "~/cosmic_reconstruction_at_nersc/c_ptychocam/ptychocam_reconstruction.sh"
```

Comment:
This has a ~ indicating that it's a home directory? Whose home directory is it? Can this script be run by someone other than you?